mmv1/third_party/terraform/services/resourcemanager/resource_google_project.go

```diff
@@ -227,7 +227,13 @@ func resourceGoogleProjectCreate(d *schema.ResourceData, meta interface{}) error
 			return errwrap.Wrapf("Error enabling the Compute Engine API required to delete the default network: {{err}} ", err)
 		}
 
-		if err = forceDeleteComputeNetwork(d, config, project.ProjectId, "default"); err != nil {
+		err = forceDeleteComputeNetwork(d, config, project.ProjectId, "default")
+		// Retry if API is not yet enabled.
+		if err != nil && transport_tpg.IsGoogleApiErrorWithCode(err, 403) {
+			time.Sleep(10 * time.Second)
```

Comment:
Noting for future reviewers: Normally we would want to send the request in a retry loop using our [standard retry predicates](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/transport/error_retry_predicates.go#L23)

But forceDeleteComputeNetwork actually involves multiple steps, so a single sleep retry should be fine in this case.

Reply:
Why are we adding the retry here instead of adding retries inside forceDeleteComputeNetwork?

Reply:
I thought about that, but the 403's we're trying to retry on here occur right after project creation. If anything wanted to reuse the function (this is the only use so far), I thought it would be better to keep this case contained to `google_project`. Open to other ideas though.